src/lib/Event/Listener/LoginListener.php

```diff
@@ -18,6 +18,7 @@
 use Symfony\Component\Security\Core\Authentication\Token\TokenInterface;
 use Symfony\Component\Security\Core\Authorization\AuthorizationCheckerInterface;
 use Symfony\Component\Security\Http\Event\InteractiveLoginEvent;
+use Symfony\Component\Security\Http\SecurityEvents;
```

Comment:
This class is not used in new code. Why is it imported and why automatic code checks (php-cs-fixer) didn't catch this unused import?

Reply:
I checked which events are available under the SecurityEvents and I forgot to remove it